spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskControllerTests.java

```diff
@@ -334,28 +334,28 @@ void saveCompositeTaskWithParameters() throws Exception {
 		assertThat(myTask2.getName()).isEqualTo("myTask-t2");
 	}
 
-	@ParameterizedTest
-	@ValueSource(strings = {"search", "taskName"})
-	void findTaskNameContainsSubstring(String taskNameRequestParamName) throws Exception {
+	@Test
+	void findTaskNameContainsSubstring() throws Exception {
+		final String TASK_NAME_REQUEST_PARAMETER = "taskName";
```

Comment:
**[TinyNit]** Please remove the `final` keyword as we only use them where it adds value (like immutable fields etc..)